components/identities/availableIdentities.tsx

```diff
@@ -214,34 +219,29 @@ const AvailableIdentities = ({ tokenId }: { tokenId: string }) => {
                   ADD IDENTITIES
                 </button>
               </div>
-              <div className={styles.containerIdentity}>
-                <>
-                  <div className={styles.identityBox}>
-                    <IdentityCard
-                      identity={identity}
-                      tokenId={tokenId}
-                      isOwner={isOwner}
-                      onPPClick={() => setIsUpdatingPp(true)}
-                      ppImageUrl={ppImageUrl}
-                    />
-                    {!hideActions ? (
-                      <IdentityActions
-                        isOwner={isOwner}
-                        tokenId={tokenId}
-                        isIdentityADomain={isIdentityADomain}
-                        identity={identity}
-                        hideActionsHandler={hideActionsHandler}
-                      />
-                    ) : (
-                      minting && <IdentityActionsSkeleton />
-                    )}
-                  </div>
-                  <IdentityWarnings
-                    isIdentityADomain={isIdentityADomain}
-                    identity={identity}
-                  />
-                </>
-              </div>
+            </div>
+            <div
+              className={` flex justify-center items-center ${styles.CardContainer}`}
+            >
+              <IdentityCard
+                identity={identity}
+                tokenId={tokenId}
+                isOwner={isOwner}
+                onPPClick={() => setIsUpdatingPp(true)}
+                ppImageUrl={ppImageUrl}
+              />
+            </div>
+            <div className=" min-w-[280px]">
+              {hideActions ? (
```

Comment:
It should be the opposite: `!hideActions`, or rename it `showActions`. Because currently, when hideActions === true, you show them, it's the opposite of what it means in english